frontend/__tests__/utils/extractModelAndProvider.test.ts

```diff
@@ -59,9 +59,9 @@ describe("extractModelAndProvider", () => {
       separator: "/",
     });
 
-    expect(extractModelAndProvider("claude-3-5-sonnet-20240620")).toEqual({
+    expect(extractModelAndProvider("claude-3-5-sonnet-20241022")).toEqual({
       provider: "anthropic",
-      model: "claude-3-5-sonnet-20240620",
+      model: "claude-3-5-sonnet-20241022",
```

Comment:
We want to remove the older Sonnet from the UI?

Reply:
I think this is test case - no?

Reply:
Ah! You're right!